content/docs/user-guide/packaging/windows-release-builds.md

```diff
@@ -0,0 +1,131 @@
+
+
+## Create a project game release layout
+
+You can create a project game release layout by using a non-monolithic or monolithic build. A monolithic release layout only contains files that are necessary to run the project, such as the Game Launcher, bundled content, and a minimal set of dependent `.dll` files, if any. A non-monolithic release layout contains the same files needed to run the project, as well as the objects that you use for development such as tools, shader compilers, and all `.dll` files.  Non-monolithic releases are typically much larger than monolithic releases. On some platforms that don't allow loading shared objects, you must use a monolithic release.
+
+In this step, you will use CMake to build either a monolithic or a non-monolithic project and specify the `INSTALL` target and `release` configuration. This handles several tasks:
+
+- Creates an `install` directory that will contain your release layout.
+
+- Bundles your game content into an `engine.pak` file by packaging all of the product assets in your `Cache/pc` directory.
+
+- Builds the Game Launcher and other supporting files.
+
+{{< note >}}
+The Game Launcher may fail to load your assets if the `engine.pak` file is over 4GB. We suggest that you clean up and reprocess your assets to reduce the file size of your bundled content.
+{{< /note >}}
+
+### Monolithic project
+
+1. Set up a new CMake build directory, `windows_mono`, for your monolithic build in your project directory. We recommend keeping this build directory separate from your non-monolithic build directory. Specify a monolithic build by enabling the `-D` option, `LY_MONOLITHIC_GAME`.
+
+    ```cmd
+    cmake -B build/windows_mono -S. -G "Visual Studio 16" -DLY_3RDPARTY_PATH=C:\o3de-packages -DLY_MONOLITHIC_GAME=1
+    ```
+
+2. Run the CMake build configuration in your monolithic build directory. This generates a Game Launcher and packages your project's assets in the `Cache` directory into `engine.pak`.
+
+    ```cmd
+    cmake --build build/windows_mono --target INSTALL --config release
+    ```
+
+The result is a project release install directory that's located at `install/bin/Windows/release/Monolithic`.
+
+
+### Non-monolithic projects
+
+1. Run the CMake build configuration in your non-monolithic build directory, `build/windows_vs2019`.
+
+    ```cmd
+    cmake --build build/windows_vs2019 --target INSTALL --config release
+    ```
+
+The result is a project release install directory that's located at `install/bin/Windows/release/Default`.
+
+
+## Run the game launcher
+
+Now that you've packaged your assets and built your project game release layout, you are ready to run your project's Game Launcher.
+
+1. Open a command line window and change to your project's release build install directory.
+
+2. Run the Game Launcher. Supply the path to the level that the project starts with. This opens the Game Launcher and loads the starting level.
```

Comment:
I don't see a path to the level in the example below. Seems like this would be an optional parameter. Do you want to show a 2nd example using the optional level path?

Reply:
Good catch! I am removing that sentence. We no longer have to supply the path to a level because we set the starting level in the first step of this guide.